SETUP_GUIDE.md

```
@@ -0,0 +1,348 @@
+#### A. Create Vercel Account
+
+1. Go to [vercel.com](https://vercel.com)
+2. Sign up with your GitHub account
+3. Verify your email address
+
+#### B. Import Your Repository
+
+1. In Vercel dashboard, click **"New Project"**
+2. Import your `GameOne` repository from GitHub
+3. Configure project settings:
+   - **Framework Preset**: Next.js
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)